src/main/java/seedu/address/model/widget/WidgetModelManager.java

```diff
@@ -10,13 +12,17 @@
  * the filtered client list.
  */
 public class WidgetModelManager implements WidgetModel {
+    // The logger is to log the
+    private static final Logger logger = LogsCenter.getLogger(WidgetModelManager.class);
     private WidgetObject widgetObject;
 
-    private WidgetModelManager() {}
+    private WidgetModelManager() {
+        widgetObject = new WidgetObject();
+    }
 
     static WidgetModelManager initWidget() {
+        logger.info("Initialising WidgetModelManager...");
         WidgetModelManager view = new WidgetModelManager();
```

Comment:
This variable doesn't seemed to be used

Reply:
This is being called but my formatting is not obvious.

Reply:
Where is this `view` variable used ah

Cuz right now its just being set to WidgetModelManager but on the next line you return a new WidgetModelManager?